src/Tizen.NUI.Components/Controls/Navigation/AppBar.cs

```diff
@@ -182,9 +184,45 @@ public View NavigationContent
             }
         }
 
+        /// <summary>
+        /// Title text of AppBar.
+        /// SetTitle sets title text to the default title content.
+        /// If TitleContent is not TextLabel, then Title does not set title text of the TitleContent.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public string Title
+        {
+            get
+            {
+                if (TitleContent == null)
+                {
+                    return null;
+                }
+
+                if (TitleContent is TextLabel)
+                {
+                    return ((TextLabel)TitleContent).Text;
+                }
+
+                return null;
+            }
```

Comment:
```suggestion
            get
            {
                if (TitleContent is TextLabel label)
                {
                    return label.Text;
                }
                return null;
            }
```

Reply:
Thank you for your comment.
I have updated Title property as you suggested below.